communication/slack-config/restrictions.yaml

```diff
@@ -27,6 +27,11 @@ restrictions:
   - path: "sig-network/*.yaml"
     channels:
       - "^sig-network-.*$"
+  - path: "sig-node/*.yaml"
+    channels:
+      - "^sig-node"
```

Comment:
Do we want this to be `^sig-node-.*$` like the others (since there are othrr channels such as `sig-node-bug-scrub`, `sig-node-kmm`, etc.).

Reply:
I think `"^sig-node"` is fine (matches any name that starts with "sig-node") and is shorter. SIG testing also uses `"^sig-testing"` below.

If it wasn't okay, the validation of the `sig-node-bug-scrub` channel would fail.

Reply:
Ack